homeassistant/components/shelly/event.py

```diff
@@ -122,7 +122,6 @@ async def async_setup_entry(
 class ShellyBlockEvent(ShellyBlockEntity, EventEntity):
     """Represent Block event entity."""
 
-    _attr_should_poll = False
```

Comment:
Parent class `ShellyBlockEntity` sets `False` for `_attr_should_poll` so this line is not needed.